orchestrator/modules/tools/composio_tool_router.py

```
@@ -0,0 +1,205 @@
+
+    Returns:
+        ComposioToolRouter instance
+    """
+    from core.models.composio_cache import AgentAppAssignment
+
+    # Get agent's assigned apps
+    assignments = (
+        db_session.query(AgentAppAssignment)
+        .filter(
+            AgentAppAssignment.agent_id == agent_id,
+            AgentAppAssignment.is_active == True,
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `AgentAppAssignment.is_active:` for truth checks

Replace with `AgentAppAssignment.is_active`

(E712)